src/test/java/com/google/jenkins/plugins/computeengine/integration/ComputeEngineCloudMultipleMatchingConfigurationsIT.java

```diff
@@ -122,7 +119,6 @@ public void testMultipleLabelsProvisionedWithLabels() throws Exception {
   private void checkOneNode(PlannedNode plannedNode, String desc)
       throws InterruptedException, java.util.concurrent.ExecutionException {
     String name = plannedNode.displayName;
-    plannedNode.future.get();
```

Comment:
What's the reasoning for removing this part?

Reply:
The thing that we care about most in this test is whether both configurations were used, which can be tested without waiting for the instances to fully launch by looking at the configuration of each planned node. This also speeds up this test, especially in the windows case.